### PR TITLE
Rewritten Server Restart Timer

### PR DESCRIPTION
**Pull request by TrapinchO**
1) rewritten the restart timer because it didn't seem to work
2) Changed 2 words in the settings so it is easier to find

### CODE REVIEW

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java [hunk 1]
```diff
@@ -592,6 +573,23 @@ else if (messageText.matches("[a-zA-Z0-9_ ]{1,19} has given you 20% resistance a
             }
             return;
         }
+
+        // Server restart message handling
+        Matcher matcher_restart = SERVER_RESTART_PATTERN.matcher(messageText);
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: Snake case and camel case for variable names on the same line 🤢

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java [hunk 2]
```diff
@@ -592,6 +573,23 @@ else if (messageText.matches("[a-zA-Z0-9_ ]{1,19} has given you 20% resistance a
...
+        if (matcher_restart.find()) {
+            if (OverlayConfig.ConsumableTimer.INSTANCE.showServerRestart) { // if you want the timer
+                int seconds = Integer.parseInt(matcher_restart.group(1));
+
+                if (matcher_restart.group(2).equals("minute") || matcher_restart.group(2).equals("minutes")) { // if it is in minutes
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: Check for minutes first so you'll leave the if statement earlier more often

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: What? It is the same as the rest (or more exactly like the 2 above)

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 2 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: How exactly?

Like do mins and divide by 60 when seconds?

Comment (HighCrit) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: Swap "minute" and "minutes"

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (HighCrit) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: Oh I see. Who did that?

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 2 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: ok, will do

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: I have no idea, I am known for not doing original code

Comment (HighCrit) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: It was you 🕵️ https://github.com/Wynntils/Wynntils/commit/a7006c9fa844245222d0098f07c00abfbc064b68

Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: lol

ok, should I rename it to "matcherRestart" or something?

Comment (HighCrit) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: restartMatcher would be better imo

Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: ok, I will rename the other ones as well

**Review by coehlrich — APPROVED** [review]

**Pull request merged**